doc/pilight-debug.1

```diff
@@ -0,0 +1,53 @@
+.TH pilight-debug 1 "3 December 2016" "7.0" "pilight 7.0"
+.SH NAME
+pilight-debug \- Interpret 433.92 MHz streams
```

Comment:
This is not entirely true. pilight-debug tries to decode incoming raw streams into logical pulsetrains.